CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# CDK Generative AI Constructs V0.1.20 (2024-01-10)
+
+Based on CDK library version 2.118.0
```

Comment:
```suggestion
Based on CDK library version 2.114.0
```

Reply:
done in [e8ebbf5](https://github.com/awslabs/generative-ai-cdk-constructs/pull/180/commits/e8ebbf58d89f9699465624722f45c3eb9401228f)